keanu-python/keanu/plots/traceplot.py

```diff
@@ -5,11 +5,11 @@
     pass
 
 import numpy as np
-from keanu.vartypes import sample_types, numpy_types
+from keanu.vartypes import sample_types, numpy_types, vertex_label_types
 from typing import Any, List
 
 
-def traceplot(trace: sample_types, labels: List[str] = None, ax: Any = None, x0: int = 0) -> Any:
+def traceplot(trace: sample_types, labels: List[vertex_label_types] = None, ax: Any = None, x0: int = 0) -> Any:
```

Comment:
As discussed - I'm not sure adding a new VertexLabel object is actually a good thing for the Python API.

Reply:
Removed. Kept the old `VertexLabel` but made it pseudo-private (now `_VertexLabel`) since it's being used as a parameter for other stuff.